### DISCUSSION TITLE
timestr values appear to have changed

### DISCUSSION THREAD

**Discussion by kpnobvious** [jertel/vuegraf#21] [General]
Good morning, earlier this morning around 6AM eastern, i noticed values stopped recording with the following error:

Traceback (most recent call last):
  File "src/vuegraf2.py", line 150, in <module>
    tmpStartingTime = datetime.datetime.strptime(timeStr, '%Y-%m-%dT%H:%M:%S.%fZ')
  File "/usr/lib/python3.7/_strptime.py", line 577, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/lib/python3.7/_strptime.py", line 362, in _strptime
    data_string[found.end():])
ValueError: unconverted data remains: Z

It looks like the time format now has an extra Z appended to it
2021-02-15T11:08:48.81664Z**Z**

I just fixed it by changing the value of timestr to remove the addition of the extra Z.  I'm not sure if this is from pyemvue or Emporia just changed something again.

Thank you!

**Comment by jertel** [comment]
That's interesting. I just checked my vuegraf process and it's not having this issue. I'll add a check to the code to see if the Z exists before appending it.

**Reply by kpnobvious** [reply]
awesome, thanks.  Funny enough it also just stopped.
